WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionButton.swift

```diff
@@ -0,0 +1,124 @@
+import UIKit
+
+extension ReaderSortingOption {
+
+    var localizedDescription: String? {
+        // TODO: check if strings are localized
+        switch self {
+        case .date:
+            return NSLocalizedString("Recent", comment: "Sorting option description")
+        case .popularity:
+            return NSLocalizedString("Popular", comment: "Sorting option description")
+        case .noSorting:
+            return nil
+        }
+    }
+
+    var image: UIImage? {
+        switch self {
+        case .date:
+            // TODO: use proper icon
+            return .gridicon(.calendar)
+        case .popularity:
+            // TODO: use proper icon
+            return .gridicon(.lineGraph)
+        case .noSorting:
+            return nil
+        }
+    }
+}
+
+class ReaderSortingOptionButton: UIControl {
+    var sourceView: UIView {
+        return chevronView
+    }
+
+    private lazy var iconView: UIImageView = {
+        let view = UIImageView()
+        view.tintColor = UIColor(light: UIColor(hexString: "4D4D4D"), dark: UIColor(hexString: "BFBFBF"))
```

Comment:
is it necessary to define custom colors here? I would think we can rely on system colors.

Reply:
I used the hex values from Figma. Do you know what system colors would those be?
I tried some of the system gray colors and they do not match the ones in Figma.
In Figma for light mode it says "Button / Black Fill / Gray Outline" and hex 4D4D4D, for dark "Button / White Fill / Gray Outline" and hex BFBFBF.

Reply:
Maybe @mbshakti can give us some guidance? Not a blocker at all, but I think it would be better to use system colors as much as we can.

Reply:
I'd say it's ok for now to leave them as they are, let's maybe just put a TODO to double check if we can use system colors?

Reply:
Also can you add the values to `Constants`?